panther_manager/src/main.cpp

```diff
@@ -25,8 +25,11 @@ int main(int argc, char ** argv)
   auto manager_bt_node = std::make_shared<panther_manager::ManagerBTNode>("manager_bt_node");
   manager_bt_node->Initialize();
 
+  rclcpp::executors::MultiThreadedExecutor executor;
```

Comment:
MultiThreadedExecutor without specified proper callback groups is in practice SingleThreadedExecutor. I assume that changes based on our latest discussion have not yet been implemented.